src/transformState.js

```diff
@@ -6,6 +6,28 @@
  */
 function transformState(state, actions) {
   // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        Object.assign(state, action.extraData);
+        break;
+
+      case 'clear':
+        for (const key of Object.keys(state)) {
+          delete state[key];
+        }
+        break;
+
+      case 'removeProperties':
+        for (const data in action.keysToRemove) {
```

Comment:
[checklist](https://github.com/mate-academy/js_stateful-object/blob/master/checklist.md) 1st item, `keysToRemove` is an array